.gitignore

```diff
@@ -1,18 +1,25 @@
+# General
+# - directories
+__pycache__/
 /.idea/
 /.run/
 bin/
 cache/
 semantic.cache/
-
+# - patterns
 *.code-workspace
 *.tmp
 *.tmp.obo
 *.tmp.owl
 *.tmp.json
 *.db
+.vscode/*
 
+# - static
 .DS_Store
 
+# Specific files, patterns, and directories
+*relation-graph*
```

Comment:
## `relation-graph` ignore
After running to create SemanticSQL DBs, I was surprised to see it wanted to commit one of the relation-graph intermediaries (which ideally would be deleted automatically). So I added it to the `.gitignore`.